src/Microsoft.Diagnostics.EventFlow.Outputs.Oms/OmsOutput.cs

```diff
@@ -157,7 +157,8 @@ public async Task SendEventsAsync(IReadOnlyCollection<EventData> events, long tr
         private string BuildSignature(string message, string dateString)
         {
             string dateHeader = $"{MsDateHeaderName}:{dateString}";
-            string signatureInput = $"POST\n{message.Length}\n{JsonContentId}\n{dateHeader}\n{OmsDataUploadResource}";
+            byte[] messageInBytes = Encoding.UTF8.GetBytes(message);
```

Comment:
Minor perf improvement suggestion: swap lines 119 and 121 above and use `StringContent.TryComputeLength()` (will always succeed for StringContent) to compute the message length, then pass it as a (new) parameter to `BuildSignature`. This way we UTF8-encode the message just once instead of twice.

Reply:
@karolz-ms thanks, unless I missed anything TryComputerLength() is an internal method I'm unable to use? I've therefore used content.Headers.ContentLength instead presumably it calls TryComputerLength internally? Feel free to make any further changes/suggestions if required. thanks.

Reply:
@lnhzd sorry, my mistake, did not look at the signature carefully enough--but you found an alternative way to implement the suggestion, great job!